### PR TITLE
feat: Add Login and Signup Pages

### PR DESCRIPTION
**Pull request by Naman-Saxena1**
# Added Login and Signup Page, functionality will be added later using Backend
Login Page Preview : https://deploy-preview-3--bookztron-dev-branch.netlify.app/login
Signup Page Preview : https://deploy-preview-3--bookztron-dev-branch.netlify.app/signup

Files to review : 
1. Login.jsx
2. Signup.jsx
3. UserAuth.css

## Login Page
Capture2

## Signup Page
Capture3

### CODE REVIEW

**Review by sri-rishi — APPROVED** [review]
Great work, couldn't find any issue, feel free to merge from my side

File: src/Pages/AuthenticationPages/UserAuth.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
+body
+{
+    height: 100vh;
+}
+
+.user-auth-content-container
+{
+    /* max-height: fit-content; */
```
Comment (varun1233) [inline src/Pages/AuthenticationPages/UserAuth.css:8]: Remove Commented code if not required

**Review by varun1233 — APPROVED** [review]
Code looks neat and good. Good to go from my side. 
Just added a small suggestion to remove commented code.

File: src/Pages/AuthenticationPages/UserAuth.css (hunk 1 above)
Comment (Naman-Saxena1) [reply src/Pages/AuthenticationPages/UserAuth.css:8]: Done.

**Pull request merged**